lib/bokken/birthday_notifier.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Bokken.BirthdayNotifier do
+  @moduledoc """
+  Executes a task periodically to notify Ninjas on their birthday.
+  """
+
+  alias Bokken.Accounts
+  alias BokkenWeb.EventsEmails
+  import Bokken.Events.EventAdmin
+
+  def notify_ninja_birthday do
```

Comment:
```suggestion
  def send_happy_birthday_ninjas do
```

I think it is better naming because the ninjas already know it's their birthday :D